features/steps/template_steps.py

```diff
@@ -19,6 +20,21 @@ def when_we_run_create_content_task(context):
 def then_next_run_is_on_monday(context, time):
     data = get_json_data(context.response)
     next_run = parse_date(data.get('next_run'))
+    fmt = '%H:%M:%S'
+
+    try:
+        # assume time is set in given time zone
+        tz = pytz.timezone(data.get('schedule').get('time_zone'))
+    except:
```

Comment:
catch all except may hide errors, would be better a data['schedule']['time_zone'] and except(KeyError) IMHO